docs/container.md

```diff
@@ -0,0 +1,69 @@
+# Container Usage
+
+The project operates GHA workflows to build and publish two OCI images to
+GHCR, `mlflow-cratedb` and `ml-runtime`.
+
+- `ghcr.io/crate-workbench/mlflow-cratedb`
+  Includes the amalgamated Mlflow for CrateDB adapter, ready to run.
+  Effectively, it contains the same packages as if you installed them with
+  `pip install 'mlflow-cratedb'`
+
+- `ghcr.io/crate-workbench/ml-runtime`
+  Includes a few popular machine learning libraries and other software
+  to support your experiments. Effectively, it contains the same packages
+  as if you installed them with `pip install 'mlflow-cratedb[examples]'`.
+
+For general usage information, please refer to the [handbook](./handbook.md).
+For building your own images, see the [development documentation](./development.md).
+On GHCR, other than `latest` images for releases, there are also images for
+each PR, as well as `nightly` ones.
+
+
+## Docker and Podman
+
+### Standalone
+
+In order to instruct MLflow to submit the experiment metadata directly to CrateDB,
+configure the `MLFLOW_TRACKING_URI` environment variable to point to your CrateDB
+server. When running the experiment within a container, instead of using `localhost`,
+you will need to address the CrateDB container by name instead.
+
+```shell
+docker run --rm -it --link=cratedb \
+  --env="MLFLOW_TRACKING_URI=crate://crate@cratedb:4200/?schema=mlflow" \
+  --env="CRATEDB_HTTP_URL=http://crate@cratedb:4200/?schema=doc" \
+  ghcr.io/crate-workbench/ml-runtime python /opt/ml/bin/tracking_dummy.py
+```
+
+### Tracking Server
+
+When running the MLflow Server within a container, you will need to address the
+CrateDB container by name instead of using `localhost`.
+```shell
+export CRATEDB_SQLALCHEMY_URL="crate://crate@cratedb:4200/?schema=mlflow"
+```
+
+Start the MLflow Server, linking it to the other container running CrateDB,
+and exposing its HTTP port 5000 to your machine, serving the HTTP API
+and the web UI, see http://localhost:5000/.
+```shell
+docker run --rm -it --name=mlflow --link=cratedb --publish=5000:5000 \
+  ghcr.io/crate-workbench/mlflow-cratedb mlflow-cratedb server \
+  --backend-store-uri="${CRATEDB_SQLALCHEMY_URL}" --host=0.0.0.0 \
+  --gunicorn-opts="--log-level=debug"
+```
+
+Start an experiment program which needs to access both CrateDB and MLflow,
+for storing data and recording experiment metadata, linking it to both
+other containers.
+```shell
+docker run --rm -it --link=cratedb --link=mlflow \
+  --env="MLFLOW_TRACKING_URI=http://mlflow:5000" \
+  --env="CRATEDB_HTTP_URL=http://crate@cratedb:4200/?schema=doc" \
+  ghcr.io/crate-workbench/ml-runtime python /opt/ml/bin/tracking_merlion.py
+```
+
+
+## Kubernetes
+
+Todo.
```

Comment:
It would be sweet if someone could support us here, to provide basic configuration snippets how such a thing would best be deployed on Kubernetes.

Unfortunately, I don't know too much about it. However, those details come to mind:

- Just add YAML?
- Use Helm?

/cc @WalBeh, @SStorm 


---

![image](https://github.com/crate-workbench/mlflow-cratedb/assets/453543/4599356f-6d07-43ed-a5bc-1b204f9ad849)